src/PropertiedObject.h

```diff
@@ -11,6 +11,7 @@ class LuaManager;
 class PropertiedObject {
 public:
 	PropertyMap &Properties() { return m_properties; }
+	const PropertyMap &Properties() const { return m_properties; }
```

Comment:
It's called from a method which is const and so the `Properties()` method must also be const.